modules/aws_k8s_service/tf_module/variables.tf

```diff
@@ -200,3 +200,8 @@ variable "tolerations" {
 variable "cron_jobs" {
   default = []
 }
+
+variable "timeout" {
+  type    = number
+  default = 300
```

Comment:
default to 600 here too?

Reply:
Or rather, not have the default here. Make this a required field in terraform